eng/common/scripts/copy-docs-to-blobstorage.ps1

```diff
@@ -231,7 +231,7 @@ function Upload-Blobs
 }
 
 
-if ((Get-ChildItem -Path Function: | ? { $_.Name -eq $PublishGithubIODocsFn }).Count -gt 0)
+if (Test-Path Function:PublishGithubIODocsFn)
```

Comment:
I like this improvement it makes these checks a lot cleaner.

Reply:
```suggestion
if (Test-Path "Function:$PublishGithubIODocsFn")
```

Reply:
This actually does not work. It always returns $True

Reply:
I believe your current code will always return false. From my local testing I think you need to add the quotes and have the "$PublishGithubIODocsFn" expand the name of the function we are checking.

Reply:
I thought I returned to the original logic. Even with the quotes this does not work because `$PublishGithubIODocsFn` is actually a variable.

Reply:
Can you please test it again as I was able to get it to work with quotes because the variable would result in a full string that would be the function location that Test-Path would be able to understand.

Reply:
It always returns true. Even when the variable is not in scope.